dart/packages/fury/lib/src/resolver/ref/ser_ref_resolver.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:fury/src/const/ref_flag.dart';
+import 'package:fury/src/resolver/impl/ref/ser_map_resolver.dart';
+import 'package:fury/src/resolver/impl/ref/ser_no_ref_resolver.dart';
+
+typedef SerRefRes = ({RefFlag refFlag, int? refId});
```

Comment:
`Res` can be recognized as `Resource`, we shoudl avoid such abbr in whole codebase. How about `SerializationRefMeta`?

I suggest to avoid all uncommon abbr in this PR. We could take https://github.com/google/json_serializable.dart/blob/master/json_serializable/lib/src/json_serializable_generator.dart as an example. 

Fury favors clarity over brevity in naming conventions.